template/dogfood/flake.nix

```diff
@@ -0,0 +1,20 @@
+{
+  description = "Description for the project";
+
+  inputs = {
+    nixpkgs_22_11.url = "github:NixOS/nixpkgs/nixos-22.11";
```

Comment:
23.11 is targeting release by EOM and 23.05 has been available for quite a while too, shouldn't this be updated to something newer?

Reply:
It's just an example of specifying a special version of `nixpkgs`

Reply:
Just a bystander here so you can skip the feedback if you prefer, but I would expect a `template` output (as opposed to a documentation-style prose example, which can be more relaxed) to produce a well-formed, actionable flake that doesn't include any subjectively-bad defaults.

Reply:
fixed